### DISCUSSION TITLE
Github Pages is Not Loading Specific Folders

### DISCUSSION THREAD

**Discussion by AyrtonB** [community/community#22161] [Other Feature Feedback, Questions, & Ideas]
<p>Hi all,</p>
<p>I’m trying to use github-pages to host an online version of a spatial programming course, however the static files for stuff like CSS and JS aren’t loading properly.</p>
<p>e.g. <a href="https://esda-spatial.github.io/Course-Site/tutorials/03%29%20Working%20with%20Vector%20Data.html" rel="noopener nofollow ugc">https://esda-spatial.github.io/Course-Site/tutorials/03)%20Working%20with%20Vector%20Data.html</a></p>
<p><a href="https://i.stack.imgur.com/aa2vj.png" rel="noopener nofollow ugc"><img src="https://i.stack.imgur.com/aa2vj.png" alt="what the website currently looks like" width="690" height="388"></a></p>
<p>Whereas when I open the same file locally it appears like this:</p>
<p><a href="https://i.stack.imgur.com/2zDj1.png" rel="noopener nofollow ugc"><img src="https://i.stack.imgur.com/2zDj1.png" alt="preview of what the website should look like" width="690" height="376"></a></p>
<p>When I inspect the network tab I can see that the CSS and JS files are 404’ing when requested, and I can’t access them either - e.g. <a href="https://esda-spatial.github.io/Course-Site/_static/jquery.js" rel="noopener nofollow ugc">https://esda-spatial.github.io/Course-Site/_static/jquery.js</a>.</p>
<p>However, when I try accessing them directly from the repo everything is fine - e.g. <a href="https://raw.githubusercontent.com/ESDA-Spatial/Course-Site/master/_static/jquery.js" rel="noopener nofollow ugc">https://raw.githubusercontent.com/ESDA-Spatial/Course-Site/master/_static/jquery.js</a></p>
<p>Github pages itself has been set up to read directly from the root repository of the master branch as well.</p>
<p>Any help solving this would be much appreciated</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hey <a class="mention" href="/ayrtonb">@AyrtonB</a>, welcome to the GitHub Support Community!</p>
<p>It looks like you’ve already managed to figure this out, but I’ll leave the answer just in case anyone else comes looking for solutions to a similar problem.</p>
<p>GitHub Pages uses <a href="https://jekyllrb.com/">Jekyll</a> to build sites by default. Jekyll doesn’t include directories that start with certain special characters when building and deploying your site, including <code>_</code>:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages-and-jekyll#configuring-jekyll-in-your-github-pages-site" target="_blank" rel="noopener">About GitHub Pages and Jekyll - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you’re using Jekyll to build your site and would like to include a file or directory that Jekyll excludes by default you can do this by specifying the path to the file/folder inside the <code>include</code> variable in your site’s <code>_config.yml</code> file:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://jekyllrb.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://jekyllrb.com/docs/configuration/options/#global-configuration" target="_blank" rel="noopener">Jekyll • Simple, blog-aware, static sites</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/2439407/181069282-2caf5ea5-8936-42f6-b878-74b24d01a737.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_10x10.png"></div>

<h3><a href="https://jekyllrb.com/docs/configuration/options/#global-configuration" target="_blank" rel="noopener">Configuration Options</a></h3>

<p>The tables below list the available settings for Jekyll, and the various options (specified in the configuration file) and flags (specified on the command-line) that control them.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>If you’re <em>not</em> using Jekyll, and are building your site using pure HTML, CSS, and JS, or are using a different static site generator, you can disable Jekyll for GitHub Pages site builds altogether by creating a file called <code>.nojekyll</code> in the root of your publishing source:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://docs.github.com/assets/images/site/favicon.svg" class="site-icon" width="16" height="16">
      <a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#static-site-generators" target="_blank" rel="noopener">docs.github.com</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://docs.github.com/en/free-pro-team@latest/github/working-with-github-pages/about-github-pages#static-site-generators" target="_blank" rel="noopener">About GitHub Pages - GitHub Docs</a></h3>



  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Doing either of these should make your site available on GitHub Pages.</p>

**Reply by coder618** [reply]
Thanks, work like charm

**Comment by AyrtonB** [comment]
<p>Completely forgot about this until I just spotted the notifications whilst searching for something else just now, thank you for replying and posting the answer. Apologies for the poor etiquette.</p>
